tools/validators/instance_validator/ontology_validation.py

```diff
@@ -0,0 +1,40 @@
+def validate_type(entity: dict, entities_map):
```

Comment:
add documentation for this file refer to the ontology validator or rdf generator

Reply:
Done